src/perf/lib/PerfClient.cpp

```diff
@@ -236,6 +239,11 @@ PerfClient::Init(
             if (!UsePacing) {
                 Settings.SetPacingEnabled(UsePacing != 0);
             }
+            #ifndef _KERNEL_MODE
```

Comment:
Need to update the `if` on line 237 too.

Reply:
Updated